src/Illuminate/Support/Str.php

```diff
@@ -932,6 +932,21 @@ public static function random($length = 16)
         })($length);
     }
 
+    /**
+     * Generate a random digits with a specified number of places.
```

Comment:
If I call this function with: `Str::randomDigits(4)`, I'm not actually getting a random integer with 4 digits, I'm getting a random integer between 1000 and 9999. I might expect that '0001' or '0010' is also a possible output.

Reply:
Good point. I'm adding changes.

Reply:
@cosmastech method updated. Thanks for the notice again!